tests/functional/ubuntu/16.04/mon/initial_members/vagrant_variables.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+# DEPLOY CONTAINERIZED DAEMONS
+docker: false
+
+# DEFINE THE NUMBER OF VMS TO RUN
+mon_vms: 1
```

Comment:
Unfortunately that won't work, the way we do the provisioning requires at least 1 OSD machine to be brought up.